staff_graded/staff_graded.py

```diff
@@ -237,7 +237,7 @@ def csv_export_handler(self, request, suffix=''):  # pylint: disable=unused-argu
             cohort=cohort).write_file(buf)
         resp = Response(buf.getvalue())
         resp.content_type = 'text/csv'
-        resp.content_disposition = f'attachment; filename="{self.location}"'
+        resp.content_disposition = f'attachment; filename="{self.location}.csv"'
```

Comment:
Nit: Wow - this repo doesn't have a single test in it. Normally I would request a test but why sully our 0% code coverage? 😺